CalculatorService.py

```diff
@@ -0,0 +1,61 @@
+import PricePayload as pp
+
+class PriceCalculator:
+    """
+    A class for calculating prices and performing price checks.
+
+    Attributes:
+        None
+
+    Methods:
+        calculate_price: Calculates the price for objects from the PricePayload class.
+        check_price: Performs a check for calculated prices for objects from the PricePayload class.
+    """
+
+
+    def calculate_price (price_payload: pp.PricePayload):
+        """
+        Calculate the price for a product on Amazon based on variables of price_payload object.
+
+        Args:
+            price_payload (object): A price object from the PricePayload class.
+
+        Returns:
+            PricePayload object with calculated price.
+        """
+
+        net_price = ((price_payload.net_product_price * (1 + price_payload.margin)) + price_payload.fixed_costs + price_payload.net_delivery_cost) 
+        magic_numebr = (price_payload.vat*(1/1/(1+price_payload.vat))) + price_payload.amz_fee + price_payload.extra_perc_from_selling_price
+        calculated_price = net_price/(1-magic_numebr)
+
+        price_payload.calculated_price = calculated_price
+
+        return 
```

Comment:
do you need to write return when method is not returning anything?

Reply:
"
In Python, if a method or function does not explicitly return a value using the return keyword, it implicitly returns None. So, technically, you don't need to include a return statement if the method is not returning anything, as Python will automatically return None. However, it's often considered good practice to include a return statement, even if it's just return None, for clarity and to explicitly indicate that the function does not return any meaningful value.
"